vscode-lean4/webpack.config.js

```diff
@@ -4,10 +4,13 @@ const path = require('path');
 const webpack = require('webpack');
 const CopyPlugin = require('copy-webpack-plugin');
 
+const prodOrDev = (env) => env.production ? 'production' : 'development';
+const minIfProd = (env) => env.production ? '.min' : '';
+
 function getWebviewConfig(env) {
 	let webview = {
 		name: 'webview',
-		mode: env.production ? 'production' : 'development',
+		mode: prodOrDev(env),
```

Comment:
Since you're already cleaning up the prod/dev switch, should we use separate files for prod/dev build outputs?  It's a bit confusing that the files have a different content depending on whether you last compiled them from vscode or on the command-line.

Reply:
I made this change in `@lean4/infoview` and then started in `vscode-lean4`, but making it there seems to complicate things more than it helps since it introduces lots of `production/development` switches in code that uses the bundle (I mean even more than [these ones](https://github.com/leanprover/vscode-lean4/pull/151/commits/7bd74683a36fca23fde454c72885c8fb906198db#diff-ece3fe1d415234d756ba9dd601f6701bb51c6bed1e5d07ecc0ac4ec4796e6e57L626)), so maybe it is best to leave as-is.

Reply:
(In particular, we might want to put `extension.production.min.js` in the `.vsix` but use `extension.development.js` for extension development, and maybe also for the test runners, which adds the extra switches. Similarly for the webview bundle.)